pytorch_lightning/utilities/parsing.py

```diff
@@ -264,14 +274,14 @@ def __repr__(self):
         return out
 
 
-def _lightning_get_all_attr_holders(model, attribute):
+def _lightning_get_all_attr_holders(model: 'pl.core.lightning.LightningModule', attribute: str) -> Any:
```

Comment:
Same for the other occurrences in this file

```suggestion
def _lightning_get_all_attr_holders(model: 'pl.LightningModule', attribute: str) -> Any:
```